docs/releases/v1.9.0.md

```diff
@@ -0,0 +1,49 @@
+# AMY release v1.9.0
+
+Major AMY [v1.9.0][] version was released after [v1.8.1][] bugfix version.
+
+Many thanks to [Piotr][] who focused on code reviews.
```

Comment:
And to [Chris](https://github.com/chrismedrela/) and [Aditya](https://github.com/narayanaditya95) who did the work :P